### DISCUSSION TITLE
Threads for Twitter and LinkedIn for MMTEB

### DISCUSSION THREAD

**Discussion by imenelydiaker** [embeddings-benchmark/mteb#312] [General]
## Post for Twitter

Main Tweet by: @KennethEnevoldsen 

Quote Retweet:
- @imenelydiaker 
- https://twitter.com/Muennighoff
- (... please add ...)

Retweet:
- @imenelydiaker 

## Post for LinkedIn

Main Post by: @KennethEnevoldsen 

Repost:
 
- we should all repost @KennethEnevoldsen's post

Posts:
- New posts on the subject in a day or 2. 
- @tomaarsen 
- @imenelydiaker 

## Communities

- Discord:
  - Hugging Face
  - AINordics
  - OpenLLM Europe 🇪🇺
- Slack
  - DDSC

**Comment by Muennighoff** [comment]
Took the freedom of editing your post a bit for tracking :)

Let's collect people/communities in the post & maybe also draft our tweets there? Does one of you want to do a "main" tweet & a "main" linkedin post @KennethEnevoldsen @imenelydiaker @MartinBernstorff that others QT/RT? (I would just QT so put myself there)

**Reply by imenelydiaker** [reply]
**Draft for LinkedIn post:** (can do better, feel free to edit)

💡 We are launching contributions for the MMTEB project! 

Join our community and build embedding benchmarks for as many languages as possible! 

MMTEB is a community-led extension for MTEB, our goal is to provide all NLP communities with a massive benchmark for embeddings.

Check the contribution guidelines [here](https://github.com/embeddings-benchmark/mteb/tree/main/docs/mmteb) and join the adventure 🚀

**Reply by imenelydiaker** [reply]
I can do the main LinkedIn post (suggested post ⬆️ ) & RT

**Reply by KennethEnevoldsen** [reply]
How about:

🌐✨ Unveiling MMTEB: A Global Initiative to extend MTEB to all languages ✨🌐

We're thrilled to announce the launch of our latest project: **MMTEB** (Multilingual Massive Text Embedding Benchmark), a community-driven initiative aimed at extending the MTEB (Massive Text Embedding Benchmark) to encompass a wide array of languages from around the globe. 🚀

**Why Join Us?**
- **Contribute to Science:** Help us build the most comprehensive benchmark for text embeddings, making it as inclusive and diverse as possible and join in as a co-author at the end.
- **Enhance Your Skills:** Engage with leading NLP methodologies and contribute to a project that will be used by researchers and professionals worldwide.
- **Community Spirit:** Join a passionate group of linguists, data scientists, and NLP enthusiasts in creating a truly impactful benchmark.

**Who Can Contribute?**
Anyone with an interest in NLP, from academic researchers to hobbyists and professionals in the field. We're looking for contributions in all languages and domains to make MMTEB a truly global resource.

**How to Get Started?**
Check out our contribution guidelines [here](https://github.com/embeddings-benchmark/mteb/tree/main/docs/mmteb) to learn more about how you can participate. Whether it's adding a new language, new domain, or speeding up the benchmarks, every bit of help is welcome!

We look forward to engage with you over on the GitHub! 👩‍💻👨‍💻

**Reply by imenelydiaker** [reply]
@KennethEnevoldsen you can post this, it's a great post! I think you can even post it on [HuggingFace Posts](https://huggingface.co/posts) (there is a waitlist though 🤔). 

As @tomaarsen said ⬇️, having different posts on the same subject is better. So, I'll post something similar but not the exact same post, and we will all repost your post. Wdyt?

**Reply by tomaarsen** [reply]
If you want, feel free to enter the [waitlist](https://huggingface.co/social-post-explorers) and forward me your HF username. I can try and get you access directly - I think this would be excellent for HF Posts.

I also agree that it's a great post. I'll repost it & post a similar post myself in a day or 2 (to avoid having 2 similar posts on one profile in the same day, presumably the algorithm won't like that)

**Reply by KennethEnevoldsen** [reply]
Wonderful @MartinBernstorff and I will finish up two PRs on structure, but then next week we will do the kick-off

**Comment by MartinBernstorff** [comment]
"Influencer" reach-outs:
* Phillip Vollet

Feel free to add more below!

**Reply by KennethEnevoldsen** [reply]
@tomaarsen would this be something you would be interested in promoting as well?

You might already be aware but MMTEB is a community project for the multilingual extension of MTEB.

**Reply by tomaarsen** [reply]
Certainly, I'd be glad to help promote this. Also, in my experience, multiple posts on LinkedIn on the same topic goes further than one main post with a few reposts.

**Comment by KennethEnevoldsen** [comment]
We are live!

I created a post [here](https://www.linkedin.com/posts/kennethenevoldsen_unveiling-mmteb-a-global-initiative-activity-7184147998707236864-fkEW?utm_source=combined_share_message&utm_medium=member_desktop). Feel free to reshare and like etc.

Also please to add your posts here to make here then we will share them around

**Reply by Muennighoff** [reply]
Cool @imenelydiaker / @KennethEnevoldsen will one of you make a thread on twitter too?

Nice just saw https://twitter.com/KCEnevoldsen/status/1778392466019823661

**Reply by Muennighoff** [reply]
I think we still need to add other people that have contributed in the past here: https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md (as mentioned here https://github.com/embeddings-benchmark/mteb/pull/302#issuecomment-2037080656 cc @imenelydiaker)
I'm a bit hesitant to quote tweet/repost it while that's still WIP, as I wouldn't want them to feel excluded 🤗

**Reply by imenelydiaker** [reply]
I'm on this. I'll try to do it quickly!

**Reply by KennethEnevoldsen** [reply]
Oh have just done it

**Reply by KennethEnevoldsen** [reply]
here #344, but @imenelydiaker if you have any corrections feel free to update it.

**Reply by Muennighoff** [reply]
Amazing! QT'd your post here: https://twitter.com/Muennighoff/status/1778433270843121759
